pkg/util/mount/mount_linux.go

```diff
@@ -282,7 +282,7 @@ func readProcMounts(mountFilePath string, out *[]MountPoint) (uint32, error) {
 }
 
 func readProcMountsFrom(file io.Reader, out *[]MountPoint) (uint32, error) {
-	hash := adler32.New()
+	hash := fnv.New32a()
```

Comment:
This method is used in kubelet to make sure `/proc/mounts` is steady.
 
It reads contents of `/proc/mounts` for multiple times and hash all results to see if any difference happens. There's no cached value of this hash result and it is only be used to compare with the result from same method. So I assume it is safe to make this change.

See: https://github.com/kubernetes/kubernetes/blob/master/pkg/util/mount/mount_linux.go#L252-L271